### PR TITLE
Setup hello world

### PR DESCRIPTION
**Pull request by Khaled-AlKhateeb**
## Tasks Done in this PR: ✔️ 

- Create a rails app by running the command `rails new -j webpack my_rr_app`
- Install needed dependencies such as react, redux, and babel.
- Configure the file  `babel.config.js`.
- Generate and set up the Root controller, view, and route.
- Update webpack.config.js to include the babel-loader.
- Set up Postgres database.
- Create the Greeting model with 5 different greeting messages stored.
- Create the API endpoint that selects a random greeting.
- Create the App component with react-router and render it in the root view.
- Create the Greeting component that will display a greeting.
- Set up redux for managing the messages state.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,217 @@
...
 
-* System dependencies
+<div align="center">
 
-* Configuration
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-* Database creation
+  <h3><b>Microverse README Template</b></h3>
```
Comment (rotimiazeez) [inline README.md]: Screenshot_20230307-145127_Chrome
- Please note that the logo provided is currently broken, you can either remove it or replace it with a logo if you have one(not microverse logo). Kindly fix it and add the project name in place of `Microverse README Template` 👍🏽

File: README.md [hunk 2]
```diff
@@ -1,24 +1,217 @@
...
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank [Microverse](https://github.com/microverseinc) for providing the project requirements and guides.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
```
Comment (rotimiazeez) [inline README.md:210]: Please ensure the MIT license link provided is valid and not a broken page 😉 
Screenshot_20230307-145140_Chrome

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi <STUDENT_MENTION> 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

File: README.md [hunk 3]
```diff
@@ -212,6 +209,6 @@ I would like to thank [Microverse](https://github.com/microverseinc) for providi
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./MIT.md) licensed.
+This project is [GNU](./LICENSE.md) licensed.
```
Comment (zieeco) [inline README.md:212]: - Hey, the licensed file is still broken as pointed out by the previous reviewer, please, ensure you fix this correctly to make the readme more professional.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Khaled-AlKhateeb ,


Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Good commit messages :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @Khaled-AlKhateeb 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**